### PR TITLE
Modify localhost_apiserver not to require preset in GET /tests

### PR DESCRIPTION
**Pull request by Fenthick**
Issue: KNTK-270

### CODE REVIEW

File: apiv6/generate_server.sh [hunk 1]
```diff
@@ -15,6 +15,7 @@ function run() {
     generate_cloud_export_server
     generate_synthetics_server
     go fmt ./...
+    git apply ./localhost_apiserver/patches/0001-Modify-localhost_apiserver-not-to-require-preset-in-.patch
```
Comment (ziemowit141) [inline apiv6/generate_server.sh:18]: Why is application of this patch required, when you also actually changed the: apiv6/localhost_apiserver/synthetics/go/api_synthetics_admin_service.go

Does it make any change it that way?

File: apiv6/go.mod [hunk 2]
```diff
@@ -3,7 +3,6 @@ module github.com/kentik/community_sdk_golang/apiv6
 go 1.15
 
 require (
-	github.com/gorilla/handlers v1.5.1
```
Comment (ziemowit141) [inline apiv6/go.mod:6]: Just out of curiosity, is this change related to the ticket? I do not see removing usage of this package in those changes.

File: apiv6/generate_server.sh (hunk 1 above)
Comment (Fenthick) [reply apiv6/generate_server.sh:18]: This way, every time the schema changes, the file will be regenerated accordingly. After regeneration is done, the patch with the fix is applied. I chose this way in order to keep the file up to date with the latest schema.

Other solution would be to disable generation of this file by adding it to `.openapi-generator-ignore`.

File: apiv6/go.mod (hunk 2 above)
Comment (Fenthick) [reply apiv6/go.mod:6]: No, I have just run `go mod tidy`

**Review by ziemowit141 — APPROVED** [review]
All my concerns got addressed

**Review by mmac-m3a — APPROVED** [review]
Fine with me, since I hope to convince Kentik engineering to remove this parameter entirely.
